tests/testdata/buf.build/grpc/go/v1.2.0/petapis/plugin.sum

```diff
@@ -1 +1 @@
-h1:tOsD3cb4cNwALEOJpfypEprBVC7eVac8wEuesWizT8c=
+h1:3fylldA9SEwekfPvkDtpVP+PntCFSjLK5iT3Xl+3Pu8=
```

Comment:
These are changing because we're now running tests with the `separate_package=true` option.